azure-iot-device/tests/iothub/pipeline/test_mqtt_topic_iothub.py

```diff
@@ -1026,6 +1026,212 @@ def test_bad_topic(self, topic):
+                    "uid": "harrypotter",
+                },
+                {
+                    "topic": "/subscriptions/resourceGroups",
+                    "subject": "/graphInstances",
+                    "$.cdid": "fakecdid",
+                    "classname": None,
+                    "dataVersion": None,
+                },
+                id="Input message topic with some custom properties not having values",
+            ),
+            pytest.param(
+                "devices/fakedevice/messages/devicebound/topic=%2Fsubscriptions%2FresourceGroups&subject=%2FgraphInstances&dataVersion&%24.cdid=fakecdid&%24.mid=e32c2285-668e-4161-a236-9f5f6b90362c&%24.cid&%24.uid&classname",
+                {"mid": "e32c2285-668e-4161-a236-9f5f6b90362c", "cid": None, "uid": None},
+                {
+                    "topic": "/subscriptions/resourceGroups",
+                    "subject": "/graphInstances",
+                    "$.cdid": "fakecdid",
+                    "classname": None,
+                    "dataVersion": None,
+                },
+                id="C2D topic with some system properties and some custom not having values",
+            ),
+            pytest.param(
+                "devices/fakedevice/modules/fakemodule/inputs/fakeinput/topic=%2Fsubscriptions%2FresourceGroups&subject=%2FgraphInstances&dataVersion&%24.cdid=fakecdid&%24.mid=e32c2285-668e-4161-a236-9f5f6b90362c&%24.cid&%24.uid&classname",
+                {"mid": "e32c2285-668e-4161-a236-9f5f6b90362c", "cid": None, "uid": None},
+                {
+                    "topic": "/subscriptions/resourceGroups",
+                    "subject": "/graphInstances",
+                    "$.cdid": "fakecdid",
+                    "classname": None,
+                    "dataVersion": None,
+                },
+                id="Input message topic with some system properties and some custom not having values",
+            ),
+        ],
+    )
+    def test_receive_topic_without_values(
+        self, topic, extracted_system_properties, extracted_custom_properties
+    ):
+        msg = Message("fake message")
+        mqtt_topic_iothub.extract_message_properties_from_topic(topic, msg)
+
+        # Validate system properties received
+        assert msg.message_id == extracted_system_properties["mid"]
+        assert msg.correlation_id == extracted_system_properties["cid"]
+        assert msg.user_id == extracted_system_properties["uid"]
+
+        # Validate system properties NOT received
+        assert msg.content_type == extracted_system_properties.get("ct", None)
+        assert msg.content_encoding == extracted_system_properties.get("ce", None)
+        assert msg.expiry_time_utc == extracted_system_properties.get("exp", None)
+
+        # Validate custom properties
+        assert msg.custom_properties == extracted_custom_properties
+
+    @pytest.mark.it("Extracts system and custom properties having none,null or empty values")
```

Comment:
>none,null or empty values [](start = 66, length = 25)

It's really just the empty string.

Perhaps

"Extracts system and/or custom properties with empty string values"?